Makefile

```diff
@@ -231,3 +233,11 @@ gardener-perf-test:
 .PHONY: gardener-aws-integration-test
 gardener-aws-integration-test:
 	./hack/ci/gardener-integration-aws-specific.sh
+
+.PHONY: module-version
+module-version:
+ifeq ($(OS_TYPE), Darwin)
+	sed -i'.bak' -e 's/VERSION/$(VERSION)/g' config/default/kustomization.yaml
```

Comment:
I also do not like the in-place update, exactly because accidental commits danger. Would much appreciate if solved with a separate template like in api-gateway module.

Reply:
Personally I also am not keen in the idea of in-place modification. For me this field should be only modified during the release to match the version that is being pushed. However I would like toczkę break kustomize and rather have separate independent make target injecting this property invoked during release. This value is meaningless during development anyway.

Reply:
the point is that during development someone may commit this and breaks release templating when its needed.

Reply:
review should catch that up I hope.

Reply:
you put too much trust into reviews, these things may get really easy overseen, I do not trust myself with reviews... always ask for second opinion 😜